lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -235,7 +235,7 @@ export default function BasicLTILaunchApp() {
     }
 
     // Don't report a submission until the URL has been successfully fetched.
```

Comment:
Not sure how to rephrase this comment "... until the content has been configured"?

Reply:
I suggest something like "Don't report a submission until we have the data needed to display the assignment content".